src/Core/MessageLogger.cpp

```diff
@@ -64,29 +66,31 @@ void MessageLogger::message(const QString &head, const QString &body, const QStr
         res += newBody;
     res += "]</span>";
 
-    box->append(res);
+    append(res);
 }
 
 void MessageLogger::info(const QString &head, const QString &body, bool htmlEscaped)
 {
-    LOG_INFO("MessageLogger Information " << INFO_OF(head) << INFO_OF(body));
+    LOG_INFO(INFO_OF(head) << INFO_OF(body));
     message(head, body, "", htmlEscaped);
 }
 
 void MessageLogger::warn(const QString &head, const QString &body, bool htmlEscaped)
 {
-    LOG_INFO("MessageLogger Warning " << INFO_OF(head) << INFO_OF(body));
+    LOG_INFO(INFO_OF(head) << INFO_OF(body));
     message(head, body, "green", htmlEscaped);
 }
 
 void MessageLogger::error(const QString &head, const QString &body, bool htmlEscaped)
 {
-    LOG_INFO("MessageLogger Error " << INFO_OF(head) << INFO_OF(body));
+    LOG_INFO(INFO_OF(head) << INFO_OF(body));
     message(head, body, "red", htmlEscaped);
 }
 
-void MessageLogger::clear()
+void MessageLogger::onAnchorClicked(const QUrl &link)
```

Comment:
Make sure that you can open external links like to website or other URLs. I am not sure but it seems this slot does nothing for normal URLs, only opens preference page links.